doc/source/user_guide/advanced.rst

```diff
@@ -179,18 +179,18 @@ on a deeper level.
 
 .. _advanced.shown_levels:
 
-Defined Levels
+Defined levels
 ~~~~~~~~~~~~~~
 
-The :class:`MultiIndex` keeps all the defined levels of an index, even
+The repr of a ``MultiIndex`` shows all the defined levels of an index, even
```

Comment:
I don't think this is a fully correct change. It is indeed shown in the repr, but here I think it was emphasized that the actual object keeps those.